src/modules/sdbus/sdbus.c

```diff
@@ -696,18 +759,22 @@ struct sdbus_ctx *sdbus_ctx_create (flux_t *h, flux_error_t *error)
 {
     struct sdbus_ctx *ctx;
 
-    if (!(ctx = calloc (1, sizeof (*ctx)))
-        || !(ctx->f_conn = sdbus_connect (h, true, retry_min, retry_max))
+    if (!(ctx = calloc (1, sizeof (*ctx))))
+        goto error_create;
+    if (sdbus_configure (ctx, flux_get_conf (h), error) < 0)
+        goto error;
+    if (!(ctx->f_conn = sdbus_connect (h, true, retry_min, retry_max))
```

Comment:
I'm probably having trouble inspecting the diff, but is the error message in `error.text` never printed if `sdbus_configure()` fails in `sdbus_ctx_create()`?

Reply:
It is printed by the caller which is in main.c.  `error` is an parameter of `sdbus_ctx_create()`.

Reply:
Ok, thanks. sorry I missed that.